internal/cmds/settings/options/options_test.go

```diff
@@ -0,0 +1,248 @@
+package options_test
+
+import (
+	"context"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/djdv/go-filesystem-utils/internal/cmds/settings/options"
+	"github.com/djdv/go-filesystem-utils/internal/cmds/settings/runtime"
+	"github.com/djdv/go-filesystem-utils/internal/parameters"
+	cmds "github.com/ipfs/go-ipfs-cmds"
+)
+
+func TestOptions(t *testing.T) {
```

Comment:
The `options` pkg used to be `settings` which used to do everything in a single package before being split up into `parameters` `options` `request`, etc.
The tests existing tests were extracted and fixed up so that they run, but were not refactored.
So they work and cover 90+% but are probably more complicated than they need to be.

Reply:
Test file has been updated since. It's probably good enough now.